src/schemas/json/unist.json

```diff
@@ -0,0 +1,158 @@
+{
+  "title": "JSON schema for unist syntax trees",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "definitions": {
+    "//": {
+      "explainer": "https://github.com/syntax-tree/unist#readme",
+      "reference": "https://github.com/syntax-tree/unist#nodes"
+    },
+    "branchDefinition": {
+      "description": "A branch is a node with one or more children.",
+      "type": "object",
+      "properties": {
+        "children": {
+          "description": "Children are direct descending nodes of another node.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/childDefinition"
+          },
+          "minItems": 1
+        },
+        "data": {
+          "$ref": "#/definitions/dataDefinition"
+        },
+        "id": {
+          "$ref": "#/definitions/idDefinition"
+        },
+        "parent": {
+          "$ref": "#/definitions/branchDefinition"
+        },
+        "position": {
+          "$ref": "#/definitions/positionDefinition"
+        },
+        "type": {
+          "$ref": "#/definitions/typeDefinition"
+        },
+        "value": {
+          "not": {}
+        }
+      },
+      "required": ["children", "type"],
+      "additionalProperties": false
+    },
+    "childDefinition": {
+      "description": "A child is a direct descending nodes of another node.",
+      "type": "object",
+      "anyOf": [
+        { "$ref": "#/definitions/branchDefinition" },
+        { "$ref": "#/definitions/literalDefinition" }
+      ]
+    },
```

Comment:
I agree it's better to make it explicit ^^.

For the `value` prop, the `boolean | string | number` is intentional? I would have gone for an "any" type:

```json
{
  "value": {
    "description": "The value field can contain any value."
  }
}
```

Reply:
I think literals are always primitives, but I’m not 100% sure about this myself. When in doubt, I think it’s best to avoid the type as you suggest.

Reply:
I don’t think additional properties should be allowed on points or positions. It’s not likely to cause any issues to define them, but if this happens it’s most certainly a mistake.

Reply:
> I think literals are always primitives, but I’m not 100% sure about this myself. When in doubt, I think it’s best to avoid the type as you suggest.

I‘d recommend going with `any`, as mentioned in the spec. There are at least some `null`s and even a few `Node`s in the wild. What a “value” is, is left unspecified!